changelog/pending/20221108--pkg--allow-case-insensitive-prop-typecheck-on-traversal.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: pkg
+  description: Allow case insensitive prop typecheck on traversal
```

For public facing changelogs, this might be a bit too obscure.

```suggestion
  scope: programgen
  description: More programs can be converted to Pulumi when using `pulumi convert`, provider bridging, and conversion tools by allowing property accesses and field names to fall back to a case insensitive lookup.
```